CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [0.15.10](https://github.com/CrowdHailer/raxx/tree/0.15.10) - 2018-09-02
```

Comment:
feel like I'm being ungrateful now. but it's  actually the third :-p

Reply:
Oh dang!! no probs!